filebeat/datadog_checks/filebeat/data/conf.yaml.example

```
@@ -1,34 +1,78 @@
+## All options defined here are available to all instances.
+#
 init_config:
 
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
+
+## Every instance is scheduled independent of the others.
+#
 instances:
 
     ## @param registry_file_path - string - required
-    ## The absolute path to the registry file used by Filebeat
+    ## The absolute path to the registry file used by Filebeat.
+    ##
     ## See https://www.elastic.co/guide/en/beats/filebeat/current/migration-registry-file.html
     #
   - registry_file_path: /var/lib/filebeat/registry
 
     ## @param stats_endpoint - string - required
     ## If Filebeat has been started with the `--httpprof [HOST]:PORT` option, then
-    ## the DD agent can gather data about the metrics Filebeat exposes to
-    ##  http://<HOST>:<PORT>/debug/vars
+    ## the Datadog agent can gather data about the metrics Filebeat exposes to  http://<HOST>:<PORT>/debug/vars.
     ##
     ## See https://www.elastic.co/guide/en/beats/filebeat/current/command-line-options.html
     ## For autodiscovery, use http://%%host%%:%%port%%/stats
     #
     stats_endpoint: http://localhost:2828/stats
 
+    ## @param normalize_metrics - boolean - optional - default: false
+    ## Normalize metric names to be prefixed with the integration name.
+    #
+    # normalize_metrics: false
+
     ## @param only_metrics - list of strings - required
-    ## This should be a list of regular expressions that stipulates
-    ## which variables should be reported to Datadog - a Filebeat metric
-    ## reported by the HTTP profiler will only be reported if it matches at
-    ## least one of those regexes.
+    ## This should be a list of regular expressions that stipulates which variables should be reported to 
+    ## Datadog - a Filebeat metric reported by the HTTP profiler will only be reported
+    ##  if it matches at least one of those regexes.
```

nit: update here and the config spec
```suggestion
    ## if it matches at least one of those regexes.
```